source/docs/software/wpilib-tools/smartdashboard/choosing-an-autonomous-program-from-smartdashboard.rst

```diff
@@ -0,0 +1,49 @@
+Choosing an autonomous program from SmartDashboard
+==================================================
+
+Often teams have more than one autonomous program, either for competitive reasons or for testing new software. Programs often vary by adding things like time delays, different strategies, etc. The methods to choose the strategy to run usually involves switches, joystick buttons, knobs or other hardware based inputs.
+
+With the SmartDashboard you can simply display a widget on the screen to choose the autonomous program that you would like to run. And with command based programs, that program is encapsulated in one of several commands. This article shows how to select an autonomous program with only a few lines of code and a nice looking user interface.
+
+Creating the SendableChooser object
+-----------------------------------
+
+.. image:: images/choosing-an-autonomous-program-from-smartdashboard/creating-sendablechooser.png
```

Comment:
This needs to be updated. Can use a remote literal include from the Hatchbot example